realm/realm.js

```diff
@@ -47,25 +50,69 @@ Symptoms.schema = {
   },
 };
 
+const REAL_PW_KEY = "REALM_PW_KEY";
+const REAM_KEY_SIZE = 64;
+
+const createKey = async () => {
+  const tmpString = base64js.fromByteArray(await generateSecureRandom(REAM_KEY_SIZE))
+  if(await Keychain.setGenericPassword(REAL_PW_KEY, tmpString))
+    return tmpString;
+  return null;
+};
+
+let createKeyInstance = null;
 const getKey = async () => {
   try {
-    const keyString = await NativeModules.EncryptionUtil.getRealmKey();
+    let keyString = null;
+    
+    if(Platform.OS == 'ios') {
+      keyString = await NativeModules.EncryptionUtil.getRealmKey();
+    } else {
+      //this strings won't show up since they are only used if biometric auth is used (which isn't here)
```

Comment:
Can this be used for iOS?

Reply:
Probably, but I haven't tested it there.